### PR TITLE
Add chaos tests readme and update bug report template

### LINKED ISSUE #6608: Need ReadMe for chaos tests
#### Describe the bug:
need readme for chaos tests to introduce 
1. why chaos tests
2. how to design chaos tests
3. how to run chaos tests

### PR DESCRIPTION
**Pull request by yanliang567**
Related issue: #6608 

Signed-off-by: yanliang567 <yanliang.qiao@zilliz.com>

### CODE REVIEW

**Review by wangting0128 — APPROVED** [review]
/lgtm

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -8,17 +8,19 @@ assignees: ''
...
+
+
+#### Actual results:
 
-#### Steps/Code to reproduce:
 
 
 #### Environment:
-- Milvus version(e.g. v1.1.0 or 8b23a93): 
+- Milvus version(e.g. v2.0.0-RC2 or 8b23a93):
```
Comment (del-zhenwu) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]: Add standalone or cluster mode?

Comment (yanliang567) [reply .github/ISSUE_TEMPLATE/bug_report.md:23]: done

**Pull request merged**